.github/workflows/build.yml

```diff
@@ -63,7 +63,9 @@ jobs:
           cache: 'gradle'
 
       - name: Build with Gradle
-        run: ./gradlew build -x test -PjdkVersion=8
+        run: |
+          ./gradlew build -x test
+          ./gradlew release -x test
```

Comment:
Is it possible to skip the test automatically when running `./gradlew release`, since the test couldn't run in release mode.